supertokens_python/recipe/passwordless/interfaces.py

```diff
@@ -176,16 +176,19 @@ def __init__(self, status: Literal['OK', 'UNKNOWN_USER_ID_ERROR',
 class DeleteUserInfoResult(ABC):
     def __init__(self, status: Literal['OK', 'UNKNOWN_USER_ID_ERROR']):
         self.status = status
+        self.is_ok = False
 
 
 class DeleteUserInfoOkResult(DeleteUserInfoResult):
     def __init__(self):
         super().__init__('OK')
+        self.is_ok = True
 
 
 class DeleteUserInfoUnknownUserIdErrorResult(DeleteUserInfoResult):
     def __init__(self):
         super().__init__('UNKNOWN_USER_ID_ERROR')
+        self.is_ok = False
```

Comment:
You do not need to set this to False since `super().__init__('UNKNOWN_USER_ID_ERROR')` already does this. Instead, you need to do `self.is_unknown_user_id_error = True`

Reply:
> You do not need to set this to False

Okay. I saw that pattern at other places in the code so I did that. 
(Example: `CreateResetPasswordWrongUserIdErrorResult`)

Fixed it.